src/MarkdownTextInput.web.tsx

```diff
@@ -236,6 +238,29 @@ const MarkdownTextInput = React.forwardRef<MarkdownTextInput, MarkdownTextInputP
       [parser, parseText, processedMarkdownStyle],
     );
 
+    const handleFormatSelection = useCallback(
+      (target: MarkdownTextInputElement, parsedText: string, cursorPosition: number, formatCommand: string): ParseTextResult => {
+        if (!contentSelection.current || contentSelection.current.end - contentSelection.current.start < 1) {
+          throw new Error('[react-native-live-markdown] invalid selection');
```

Comment:
Let's rephrase the error message to be more specific.
```suggestion
          throw new Error('[react-native-live-markdown] Trying to apply format command on empty selection');
```

Reply:
Done